### ISSUE TITLE
polyfill.io / bootcdn.net / bootcss.com / ... usages tracking

### ISSUE THREAD

**Issue by LeSuisse** [NixOS/nixpkgs#323379]
# Description

On June 25 an ongoing supply chain attack via resources served by `cdn.polyfill.io` was exposed: https://sansec.io/research/polyfill-supply-chain-attack

Yesterday more domains used by the same actor have been identified.
Currently (2024-06-29), Namecheap has put on hold the `polyfill.io` domain but the other domains are still active.

Some usages of resources served by these domains have been spotted early by the nixpkgs community by looking at the content of their `/nix/store`:
* `pandoc` (`[3.1.1; 3.1.12.3[`) #322743
* `doxygen` (`[1.9.2; 1.10.0[`, NixOS 23.11 only) #323233

In order to have a better cartography I checked the free/non broken/non insecure x86_64 packages we have in the cache for NixOS unstable b2852eb9365c6de48ffb0dc2c9562591f652242a (thanks to delroth grep-nixos-cache tool :green_heart: ). I looked for the same domains currently blocked by [uBlock Origin](https://github.com/uBlockOrigin/uAssets/blob/dad34ba26a3c76c7eaa933a1358ccd38bd27c157/filters/badware.txt#L4167-L4184).

Raw results here: https://gist.github.com/LeSuisse/05b5e3f3bf72d43f4e433778dfab486b

# Triage

I'm creating this issue to keep a public trace of the triaging work and remediation for impacted nixpkgs packages.

**WIP - This is ongoing work**

## Impacted

* [ ] `emacsPackages.org-xlatex`: [used in a template](https://github.com/ksqsf/org-xlatex/blob/f3cac5ea904e78ae2b47723fdce96e8e87edbe7c/org-xlatex.html#L16)

## Not Impacted

* `adguardhome`: filter lists
* `affine`: only used as a string in a warning message of [a JS dep](https://github.com/remix-run/react-router/blob/2f25789e60412aa5ed02442f81b315b1a2407562/packages/react-router-dom/index.tsx#L1469)
* `alice-ng`: only used as a string in a warning message of [a JS dep](https://github.com/remix-run/react-router/blob/2f25789e60412aa5ed02442f81b315b1a2407562/packages/react-router-dom/index.tsx#L1469)
* `clash-verge-rev`: usage of [V2Ray geosites.dat](https://github.com/v2fly/domain-list-community)
* `clash-verge`: usage of [V2Ray geosites.dat](https://github.com/v2fly/domain-list-community)
* `coursera-dl`: Not directly impacted but upstream README should [not advise to use `cdn.bootcss.com`](https://github.com/coursera-dl/coursera-dl/blob/10ba6b8d8c30798a45d45db2dc147edf3b455350/README.md?plain=1#L344)
* `deepin.deepin-voice-note`:usages of [`cdn.bootcdn.net` are commented](https://github.com/linuxdeepin/deepin-voice-note/blob/8b87186dd58a5b0ab85849bdfb1552257561d1f8/assets/web/index.html#L12)
* `dismap`: static detection lists [[0]](https://github.com/zhzyker/dismap/blob/bb670d2b0b26ca9f52cb25a6a5757e776d43ab5e/configs/rule.go#L522-L523) [[1]](https://github.com/zhzyker/dismap/blob/bb670d2b0b26ca9f52cb25a6a5757e776d43ab5e/configs/rule.go#L3266)
* `documenso`: [ESLint rule to not use `polyfill.io`](https://github.com/vercel/next.js/blob/b1eab7fb5ed9b0fdce96f36a0f8dc7771efaba2f/packages/eslint-plugin-next/src/rules/no-unwanted-polyfillio.ts)
* `eclipses.eclipse-jee`: used in [the README of a JS dep](https://github.com/mrmckeb/typescript-plugin-css-modules/blob/4879b0418610eff871472ffa2cc592728d84d2be/README.md?plain=1#L239)
* `eclipse-committers`: used in [the README of a JS dep](https://github.com/mrmckeb/typescript-plugin-css-modules/blob/4879b0418610eff871472ffa2cc592728d84d2be/README.md?plain=1#L239)
* `firebase-tools`: used in the [the README of a JS dep](https://github.com/matthew-andrews/isomorphic-fetch/blob/fc5e0d0d0b180e5b4c70b2ae7f738c50a9a51b25/README.md?plain=1#L57)
* `geph.cli`: probably not impacted, seems to be a static list
* `gowitness`: static list from [wappalizer](https://github.com/tunetheweb/wappalyzer/blob/e0c0009302ec8880aa24217dd7281898a9833b40/src/technologies/p.json#L2168)
* `grafana`: hit from a JS map file
* `gramma`: used in the [the README of a JS dep](https://github.com/matthew-andrews/isomorphic-fetch/blob/fc5e0d0d0b180e5b4c70b2ae7f738c50a9a51b25/README.md?plain=1#L57)
* `grafana-image-renderer`: used in the [the README of a JS dep](https://github.com/matthew-andrews/isomorphic-fetch/blob/fc5e0d0d0b180e5b4c70b2ae7f738c50a9a51b25/README.md?plain=1#L57)
* `heroic-unwrapped`: used as a string in a warning message of [a JS dep](https://github.com/remix-run/react-router/blob/2f25789e60412aa5ed02442f81b315b1a2407562/packages/react-router-dom/index.tsx#L1469)
* `hedgedoc`: used in [the sample file of JS dep](https://github.com/Azure/azure-storage-node/blob/6873387fc65bad6d577babe278be2ee2e6071493/browser/samples/sample-blob.html#L7)
* `hsd`: used in [a static list](https://github.com/Azure/azure-storage-node/blob/6873387fc65bad6d577babe278be2ee2e6071493/browser/samples/sample-blob.html#L7)

**Comment by 06kellyjac** [comment]
Are there still 2 outstanding impacted packages?
